.github/workflows/no_old_refs.yml

```diff
@@ -3,28 +3,52 @@
 #
 # Workflow starts when:
 # 1) push on master branch
+# 2) pull request is opened or synchronized or reopened
```

Comment:
When is a PR synchronized?

Reply:
https://github.community/t/what-is-a-pull-request-synchronize-event/14784

>that happens when the source branch is updated.

So simple example is force push to GitHub of sources